04 Event Emitters/assets/js/app/App.js

```diff
@@ -0,0 +1,8 @@
+(function (window) {
```

Comment:
Sorry there is any advantage of code this way instead this:

` 'use strict';
 
 var App = App || {};
 App.components = App.components || {};`